.github/workflows/test-holonix-setup.yml

```
@@ -35,3 +34,29 @@ jobs:
           done
           set -e
           nix run --version
+      - name: Test configure cache
+        env:
+          NIX_CONFIG: "access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}"
+          CACHE_PROTOCOL_OVERRIDE: https
+          CACHE_DOMAIN_SUFFIX_OVERRIDE: .cachix.org
+          CACHE_PUBLIC_KEY_OVERRIDE: holochain-ci.cachix.org-1:5IUSkZc0aoRS53rfkvH9Kid40NpyjwCMCzwRTXy+QN8=
+        run: |
+          set -x +e
+          for file in \
+              ~/.profile \
+              /etc/profile \
+              /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh \
+              /nix/var/nix/profiles/default/etc/profile.d/nix.sh
+          do
+              source $file > /dev/null 2>&1
+              if command -v nix; then break; fi
+          done
+          set -e
+
+          bash <(curl https://raw.githubusercontent.com/holochain/holochain/${{ github.sha }}/configure-cache.sh) use holochain-ci
+
+          nix_system=$(nix show-config --json | jq -r .system.value)
+
+          nix build -L --refresh -j0 \
```

would be ideal if this was cached but it depends on https://github.com/holochain/holochain/pull/2816 AFAIU

```suggestion
          nix develop --build -L --refresh -j0 \
```